packages/platform/examples/base/README.md

```diff
@@ -1,35 +1,36 @@
-# Atoms examples
-Example apps using atoms - customizable UI components to integrate scheduling into your product.
+![image](https://github.com/calcom/cal.com/assets/8019099/02705557-e6c8-4f84-bf64-b7d87682efaf)
 
-## Running examples apps locally
-
-1. Cal's backend is required to run example apps, so clone `https://github.com/calcom/cal.com` and follow setup instructions in the readme. Importantly, you need to have Google credentials setup by following [this section](https://github.com/calcom/cal.com?tab=readme-ov-file#obtaining-the-google-api-credentials) in the docs.
+<p align="center">
+    Cal.com Platform Starter Kit
+    <br />
+    <a href="https://cal.com/platform"><strong>Learn more »</strong></a>
+    <br />
+    <br />
+    <a href="https://go.cal.com/discord">Discord</a>
+    ·
+    <a href="https://cal.com">Website</a>
+    ·
+    <a href="https://github.com/calcom/cal.com/issues?q=is:issue+is:open+sort:updated-desc+label:platform">Issues</a>
+    ·
+    <a href="https://cal.com/roadmap">Roadmap</a>
+    ·
+    <a href="https://platform.cal.qa">Demo</a>
```

Comment:
@ThyMinimalDev lets try and get the example hosted under platform.cal.qa or similar